test/e2e/localnet-underlay.go

```diff
@@ -117,6 +131,19 @@ func ovsEnableVLANAccessPort(ovsNodeName string, bridgeName string, portName str
 	return nil
 }
 
+func ovsRemoveVLANAccessPort(ovsNodeName string, bridgeName string, portName string, vlanID int) error {
+	cmd := []string{
+		"kubectl", "-n", ovnNamespace, "exec", ovsNodeName, "--",
+		"ovs-vsctl", "del-port", bridgeName, portName,
+	}
+
+	if _, err := runCommand(cmd...); err != nil {
+		return fmt.Errorf("failed to remove port %s to OVS bridge %s: %v", portName, bridgeName, err)
+	}
+
+	return nil
+}
```

Comment:
We don't use and not need the vlanID to remove a port.

Reply:
Removed in previous [comment](https://github.com/ovn-kubernetes/ovn-kubernetes/pull/4863/commits/6cd4bd0fff6b0c2d1f5ed334aa6fa8ba3633fde6#r1877909978), or do you mean something else?